docs/nydusify.md

```diff
@@ -170,13 +170,37 @@ nydusify check \
   --backend-config-file /path/to/backend-config.json
 ```
 
+
+## Mount the nydus image as a filesystem
+
+![image-20230620145910421](./images/nydusify-mount.png)
+
+The nydusify mount command can mount a nydus image stored in the backend as a filesystem. Now  the  supported backend types include Registry (default backend), s3 and oss. 
+
+When using Registy as the backend, you don't need specify the `-- backend-type` option:
+
+```shell
+nydusify mount \
+  --target myregistry/repo:tag-nydus
+```
+
+Specify `--backend-type` and `--backend-config` options to mount as a filesystem:
```

Comment:
Specify `--backend-type` and `--backend-config` options to mount for other backends: